apps/dashboard/src/api/subscribers.ts

```diff
@@ -143,7 +143,10 @@ export const createSubscriber = async ({
   environment: IEnvironment;
   subscriber: Partial<CreateSubscriberRequestDto>;
 }) => {
-  const { data } = await postV2<{ data: SubscriberResponseDto }>(`/subscribers`, {
+  const queryParams = new URLSearchParams();
+  queryParams.append('failIfExists', 'true');
```

Comment:
use the new flag `failIfExists` in the Dashboard when subscriber or topic is created